src/renderers/dom/shared/ReactDOMDebugTool.js

```diff
@@ -61,6 +59,4 @@ var ReactDOMDebugTool = {
   },
 };
 
-ReactDOMDebugTool.addDevtool(ReactDOMUnknownPropertyDevtool);
```

Comment:
Why are you removing the devtool?  This appears to be going in the wrong/opposite direction.